src/Microsoft.CodeQuality.Analyzers.Exp/Core/Security/ReviewSqlQueriesForSecurityVulnerabilities.cs

```diff
@@ -105,7 +105,7 @@ public override void Initialize(AnalysisContext context)
                         var propertyReference = (IPropertyReferenceOperation)operationContext.Operation;
 
                         // We're only interested in implementations of IDbCommand.CommandText
-                        if (!propertyReference.Property.IsImplementationOfInterfaceMember(commandTextProperty))
+                        if (!propertyReference.Property.IsOverrideOrImplementationOfInterfaceMember(commandTextProperty))
```

Comment:
Filed https://github.com/dotnet/roslyn-analyzers/issues/1627 to audit other usages of `IsImplementationOfInterfaceMember`.